2019-01-08 19:01:22,287-[INFO]-[47]--my_log.py-[日志信息]:——————开始执行第1条用例：两个零相加————————
2019-01-08 19:01:22,287-[ERROR]-[57]--my_log.py-[日志信息]:断言错误0 != 1
2019-01-08 19:01:22,287-[INFO]-[47]--my_log.py-[日志信息]:本条用例不通过
2019-01-08 19:01:22,315-[INFO]-[47]--my_log.py-[日志信息]:0 + 0的值是0
2019-01-08 19:01:22,315-[INFO]-[47]--my_log.py-[日志信息]:——————本条用例执行完毕————————
2019-01-08 19:01:22,329-[INFO]-[47]--my_log.py-[日志信息]:——————开始执行第2条用例：一个零一个正数相加————————
2019-01-08 19:01:22,354-[INFO]-[47]--my_log.py-[日志信息]:0 + 5的值是5
2019-01-08 19:01:22,355-[INFO]-[47]--my_log.py-[日志信息]:——————本条用例执行完毕————————
2019-01-08 19:01:22,369-[INFO]-[47]--my_log.py-[日志信息]:——————开始执行第3条用例：一个零一个负数相加————————
2019-01-08 19:01:22,395-[INFO]-[47]--my_log.py-[日志信息]:0 + -6的值是-6
2019-01-08 19:01:22,395-[INFO]-[47]--my_log.py-[日志信息]:——————本条用例执行完毕————————
2019-01-08 19:01:22,408-[INFO]-[47]--my_log.py-[日志信息]:——————开始执行第4条用例：两个正数相加————————
2019-01-08 19:01:22,433-[INFO]-[47]--my_log.py-[日志信息]:1 + 23的值是24
2019-01-08 19:01:22,433-[INFO]-[47]--my_log.py-[日志信息]:——————本条用例执行完毕————————
2019-01-08 19:01:22,446-[INFO]-[47]--my_log.py-[日志信息]:——————开始执行第5条用例：一个正数和一个负数相加————————
2019-01-08 19:01:22,471-[INFO]-[47]--my_log.py-[日志信息]:3 + -13的值是-10
2019-01-08 19:01:22,471-[INFO]-[47]--my_log.py-[日志信息]:——————本条用例执行完毕————————
2019-01-08 19:01:22,492-[INFO]-[47]--my_log.py-[日志信息]:——————开始执行第6条用例：两个正数相加————————
2019-01-08 19:01:22,492-[ERROR]-[57]--my_log.py-[日志信息]:断言错误57 != 9
2019-01-08 19:01:22,493-[INFO]-[47]--my_log.py-[日志信息]:本条用例不通过
2019-01-08 19:01:22,531-[INFO]-[47]--my_log.py-[日志信息]:12 + 45的值是57
2019-01-08 19:01:22,531-[INFO]-[47]--my_log.py-[日志信息]:——————本条用例执行完毕————————
2019-01-08 19:01:22,547-[INFO]-[47]--my_log.py-[日志信息]:——————开始执行第7条用例：一个正数和一个负数相加————————
2019-01-08 19:01:22,576-[INFO]-[47]--my_log.py-[日志信息]:-5 + 11的值是6
2019-01-08 19:01:22,576-[INFO]-[47]--my_log.py-[日志信息]:——————本条用例执行完毕————————
2019-01-08 19:01:22,590-[INFO]-[47]--my_log.py-[日志信息]:——————开始执行第8条用例：两个负数相加————————
2019-01-08 19:01:22,618-[INFO]-[47]--my_log.py-[日志信息]:-6 + -3的值是-9
2019-01-08 19:01:22,618-[INFO]-[47]--my_log.py-[日志信息]:——————本条用例执行完毕————————
2019-01-08 19:01:22,632-[INFO]-[47]--my_log.py-[日志信息]:——————开始执行第9条用例：两个负数相加————————
2019-01-08 19:01:22,658-[INFO]-[47]--my_log.py-[日志信息]:-13 + -12的值是-25
2019-01-08 19:01:22,659-[INFO]-[47]--my_log.py-[日志信息]:——————本条用例执行完毕————————
2019-01-08 19:01:22,673-[INFO]-[47]--my_log.py-[日志信息]:——————开始执行第10条用例：两个正数相加————————
2019-01-08 19:01:22,673-[ERROR]-[57]--my_log.py-[日志信息]:断言错误123 != 1238
2019-01-08 19:01:22,673-[INFO]-[47]--my_log.py-[日志信息]:本条用例不通过
2019-01-08 19:01:22,703-[INFO]-[47]--my_log.py-[日志信息]:120 + 3的值是123
2019-01-08 19:01:22,703-[INFO]-[47]--my_log.py-[日志信息]:——————本条用例执行完毕————————
2019-01-18 13:19:58,867-[INFO]-[47]--my_log.py-[日志信息]:——————开始执行第1条用例：两个零相加————————
2019-01-18 13:19:58,867-[ERROR]-[57]--my_log.py-[日志信息]:断言错误0 != 1
2019-01-18 13:19:58,867-[INFO]-[47]--my_log.py-[日志信息]:本条用例不通过
2019-01-18 13:19:58,992-[INFO]-[47]--my_log.py-[日志信息]:0 + 0的值是0
2019-01-18 13:19:58,992-[INFO]-[47]--my_log.py-[日志信息]:——————本条用例执行完毕————————
2019-01-18 13:19:59,102-[INFO]-[47]--my_log.py-[日志信息]:——————开始执行第2条用例：一个零一个正数相加————————
2019-01-18 13:19:59,133-[INFO]-[47]--my_log.py-[日志信息]:0 + 5的值是5
2019-01-18 13:19:59,133-[INFO]-[47]--my_log.py-[日志信息]:——————本条用例执行完毕————————
2019-01-18 13:19:59,149-[INFO]-[47]--my_log.py-[日志信息]:——————开始执行第3条用例：一个零一个负数相加————————
2019-01-18 13:19:59,164-[INFO]-[47]--my_log.py-[日志信息]:0 + -6的值是-6
2019-01-18 13:19:59,164-[INFO]-[47]--my_log.py-[日志信息]:——————本条用例执行完毕————————
2019-01-18 13:19:59,180-[INFO]-[47]--my_log.py-[日志信息]:——————开始执行第4条用例：两个正数相加————————
2019-01-18 13:19:59,211-[INFO]-[47]--my_log.py-[日志信息]:1 + 23的值是24
2019-01-18 13:19:59,211-[INFO]-[47]--my_log.py-[日志信息]:——————本条用例执行完毕————————
2019-01-18 13:19:59,227-[INFO]-[47]--my_log.py-[日志信息]:——————开始执行第5条用例：一个正数和一个负数相加————————
2019-01-18 13:19:59,258-[INFO]-[47]--my_log.py-[日志信息]:3 + -13的值是-10
2019-01-18 13:19:59,258-[INFO]-[47]--my_log.py-[日志信息]:——————本条用例执行完毕————————
2019-01-18 13:19:59,274-[INFO]-[47]--my_log.py-[日志信息]:——————开始执行第6条用例：两个正数相加————————
2019-01-18 13:19:59,274-[ERROR]-[57]--my_log.py-[日志信息]:断言错误57 != 9
2019-01-18 13:19:59,274-[INFO]-[47]--my_log.py-[日志信息]:本条用例不通过
2019-01-18 13:19:59,305-[INFO]-[47]--my_log.py-[日志信息]:12 + 45的值是57
2019-01-18 13:19:59,305-[INFO]-[47]--my_log.py-[日志信息]:——————本条用例执行完毕————————
2019-01-18 13:19:59,320-[INFO]-[47]--my_log.py-[日志信息]:——————开始执行第7条用例：一个正数和一个负数相加————————
2019-01-18 13:19:59,336-[INFO]-[47]--my_log.py-[日志信息]:-5 + 11的值是6
2019-01-18 13:19:59,336-[INFO]-[47]--my_log.py-[日志信息]:——————本条用例执行完毕————————
2019-01-18 13:19:59,367-[INFO]-[47]--my_log.py-[日志信息]:——————开始执行第8条用例：两个负数相加————————
2019-01-18 13:19:59,383-[INFO]-[47]--my_log.py-[日志信息]:-6 + -3的值是-9
2019-01-18 13:19:59,383-[INFO]-[47]--my_log.py-[日志信息]:——————本条用例执行完毕————————
2019-01-18 13:19:59,399-[INFO]-[47]--my_log.py-[日志信息]:——————开始执行第9条用例：两个负数相加————————
2019-01-18 13:19:59,430-[INFO]-[47]--my_log.py-[日志信息]:-13 + -12的值是-25
2019-01-18 13:19:59,430-[INFO]-[47]--my_log.py-[日志信息]:——————本条用例执行完毕————————
2019-01-18 13:19:59,445-[INFO]-[47]--my_log.py-[日志信息]:——————开始执行第10条用例：两个正数相加————————
2019-01-18 13:19:59,445-[ERROR]-[57]--my_log.py-[日志信息]:断言错误123 != 1238
2019-01-18 13:19:59,445-[INFO]-[47]--my_log.py-[日志信息]:本条用例不通过
2019-01-18 13:19:59,477-[INFO]-[47]--my_log.py-[日志信息]:120 + 3的值是123
2019-01-18 13:19:59,477-[INFO]-[47]--my_log.py-[日志信息]:——————本条用例执行完毕————————
2019-02-12 12:38:00,547-[INFO]-[47]--my_log.py-[日志信息]:——————开始执行第1条用例：两个零相加————————
2019-02-12 12:38:00,562-[ERROR]-[57]--my_log.py-[日志信息]:断言错误0 != 1
2019-02-12 12:38:00,562-[INFO]-[47]--my_log.py-[日志信息]:本条用例不通过
2019-02-12 12:38:00,640-[INFO]-[47]--my_log.py-[日志信息]:0 + 0的值是0
2019-02-12 12:38:00,640-[INFO]-[47]--my_log.py-[日志信息]:——————本条用例执行完毕————————
2019-02-12 12:38:01,015-[INFO]-[47]--my_log.py-[日志信息]:——————开始执行第2条用例：一个零一个正数相加————————
2019-02-12 12:38:01,031-[INFO]-[47]--my_log.py-[日志信息]:0 + 5的值是5
2019-02-12 12:38:01,031-[INFO]-[47]--my_log.py-[日志信息]:——————本条用例执行完毕————————
2019-02-12 12:38:01,047-[INFO]-[47]--my_log.py-[日志信息]:——————开始执行第3条用例：一个零一个负数相加————————
2019-02-12 12:38:01,078-[INFO]-[47]--my_log.py-[日志信息]:0 + -6的值是-6
2019-02-12 12:38:01,078-[INFO]-[47]--my_log.py-[日志信息]:——————本条用例执行完毕————————
2019-02-12 12:38:01,094-[INFO]-[47]--my_log.py-[日志信息]:——————开始执行第4条用例：两个正数相加————————
2019-02-12 12:38:01,109-[INFO]-[47]--my_log.py-[日志信息]:1 + 23的值是24
2019-02-12 12:38:01,109-[INFO]-[47]--my_log.py-[日志信息]:——————本条用例执行完毕————————
2019-02-12 12:38:01,125-[INFO]-[47]--my_log.py-[日志信息]:——————开始执行第5条用例：一个正数和一个负数相加————————
2019-02-12 12:38:01,156-[INFO]-[47]--my_log.py-[日志信息]:3 + -13的值是-10
2019-02-12 12:38:01,156-[INFO]-[47]--my_log.py-[日志信息]:——————本条用例执行完毕————————
2019-02-12 12:38:01,172-[INFO]-[47]--my_log.py-[日志信息]:——————开始执行第6条用例：两个正数相加————————
2019-02-12 12:38:01,172-[ERROR]-[57]--my_log.py-[日志信息]:断言错误57 != 9
2019-02-12 12:38:01,172-[INFO]-[47]--my_log.py-[日志信息]:本条用例不通过
2019-02-12 12:38:01,219-[INFO]-[47]--my_log.py-[日志信息]:12 + 45的值是57
2019-02-12 12:38:01,219-[INFO]-[47]--my_log.py-[日志信息]:——————本条用例执行完毕————————
2019-02-12 12:38:01,234-[INFO]-[47]--my_log.py-[日志信息]:——————开始执行第7条用例：一个正数和一个负数相加————————
2019-02-12 12:38:01,265-[INFO]-[47]--my_log.py-[日志信息]:-5 + 11的值是6
2019-02-12 12:38:01,265-[INFO]-[47]--my_log.py-[日志信息]:——————本条用例执行完毕————————
2019-02-12 12:38:01,281-[INFO]-[47]--my_log.py-[日志信息]:——————开始执行第8条用例：两个负数相加————————
2019-02-12 12:38:01,297-[INFO]-[47]--my_log.py-[日志信息]:-6 + -3的值是-9
2019-02-12 12:38:01,297-[INFO]-[47]--my_log.py-[日志信息]:——————本条用例执行完毕————————
2019-02-12 12:38:01,312-[INFO]-[47]--my_log.py-[日志信息]:——————开始执行第9条用例：两个负数相加————————
2019-02-12 12:38:01,344-[INFO]-[47]--my_log.py-[日志信息]:-13 + -12的值是-25
2019-02-12 12:38:01,344-[INFO]-[47]--my_log.py-[日志信息]:——————本条用例执行完毕————————
2019-02-12 12:38:01,344-[INFO]-[47]--my_log.py-[日志信息]:——————开始执行第10条用例：两个正数相加————————
2019-02-12 12:38:01,344-[ERROR]-[57]--my_log.py-[日志信息]:断言错误123 != 1238
2019-02-12 12:38:01,344-[INFO]-[47]--my_log.py-[日志信息]:本条用例不通过
2019-02-12 12:38:01,375-[INFO]-[47]--my_log.py-[日志信息]:120 + 3的值是123
2019-02-12 12:38:01,375-[INFO]-[47]--my_log.py-[日志信息]:——————本条用例执行完毕————————
